client/src/components/DashBoard/Posts.js

```diff
@@ -0,0 +1,176 @@
+import React, { useCallback, useRef } from "react";
+import { useHistory } from "react-router-dom";
+import styled from "styled-components";
+import {
+  InfiniteLoader,
+  AutoSizer,
+  WindowScroller,
+  List,
+  CellMeasurer,
+  CellMeasurerCache,
+} from "react-virtualized";
+
+//Local
+import Post from "./Post";
+import Loader from "components/Feed/StyledLoader";
+
+// Constants
+import { mq } from "constants/theme";
+
+const cellMeasurerCache = new CellMeasurerCache({
+  fixedWidth: true,
+  defaultHeight: 380,
+});
+
+const HorizontalRule = styled.hr`
+  display: none;
```

Comment:
Sorry, this is probably out of scope and more of a nitpick of the overall React code structure/design pattern rather than the PR specifically, but I've noticed that we are basically inlining/embedding CSS styles within the JavaScript code, but I feel like CSS code should be in their own separate files, and then loaded in. Assuming we name the CSS file the same as the component, it would be easier to separate styles from component code, and make it more maintainable and reusable.

Reply:
Can you elaborate this a little more? We are using styled-components in front-end consistently in every component. I believe this is a decision to choose styled-components or to go with separated CSS files. And I assume that the decision is given at the very beginning of the project.

Reply:
Can you elaborate this a little more? We are using styled-components in front-end consistently in every component. I believe this is a decision to choose styled-components or to go with separated CSS files. And I assume that the decision is given at the very beginning of the project.

Reply:
Yeah, I'm just not seeing how that's better than a separate CSS file that can be loaded in and be able to reuse styles across multiple components. Maybe the rationale was to avoid leaking styles across components and make it easier to debug. But take my comment with a grain of salt; I am not a frontend expert.